SuccessEycaActivationActivity/handler.ts

```diff
@@ -17,10 +17,11 @@ import { ErrorResponse } from "../generated/eyca-api/ErrorResponse";
 import { ShortDate } from "../generated/eyca-api/ShortDate";
 import { UserEycaCardModel } from "../models/user_eyca_card";
 import { ActivityResult, failure, success } from "../utils/activity";
-import { extractEycaExpirationDate } from "../utils/cgn_checks";
 import { errorsToError } from "../utils/conversions";
 
 export const ActivityInput = t.interface({
+  activationDate: ShortDate,
+  expirationDate: ShortDate,
```

Comment:
ditto

Reply:
Dome in [#30fb002](https://github.com/pagopa/io-functions-cgn/pull/27/commits/30fb002e32ae2a5436af016e2f7d52633bd4017c)